website/docs/addon_core_settings.md

```diff
@@ -99,6 +99,16 @@ Applicable context filters:
 
 (This image shows use case where `render` anatomy template is used for products of families ['review, 'render', 'prerender'], `publish` template is chosen for all other.)
 
+### Filter creator profiles
+
+This provides a way how to limit list of provided Creators in Publisher based on current context. This should
+help artists decision which creator they should use.
+
+If you want to limit which creators should be shown, provide some profile, based on host name and task info.
+Put creator identifier to `Allowed Creator Identifiers`, in most cases it should be same value as resulting product type (lower cased).
+
+![core_tools_custom_staging_dir](assets/tools/core_tools_filter_creators.png)
```

Comment:
tbh, I was confused for the first glance because creator identifier looked the same as the product type. 
soon, I figure d out that this is the case in After Effects. 
Maybe, we can add one more example to show that creator identifiers are different in different Addons. 

![image](https://github.com/ynput/ayon-documentation/assets/20871534/0c7bff65-20db-4661-ac6a-9fd1e1a012d5)